Classes/common/HTTP/CDTURLSessionInterceptorContext.h

```diff
@@ -0,0 +1,39 @@
+//
+//  CDTURLSEssionFilterContext.h
+//  
+//
+//  Created by Rhys Short on 17/08/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import "CDTMacros.h"
+
+@interface CDTURLSessionInterceptorContext : NSObject
+
+@property (nonnull, readonly, nonatomic, strong) NSMutableURLRequest *request;
```

Comment:
Is there a conventional ordering for this? `nonatomic, strong, readonly` is what we've typically used, not sure where `nonnull` goes in this.